src/DataBuilder.php

```diff
@@ -775,10 +775,10 @@ public function scrub(&$data, $replacement = '*')
                 $parsedValues = array_values($parsed);
                 
                 /**
-                 * If we have at least one key/value pair (i.e. a=b) then
+                 * If we have at least two pairs (i.e. a=b&c=d) then
```

Comment:
What about cases where the query string only includes one argument i.e.: hostname.com/home?arg1=val1

I would assume that is the most common use case of the query string.